src/main/scala/com/gu/contentapi/utils/PredicateUtils.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.contentapi.utils
+
+object PredicateUtils {
+
+  implicit class RichPredicate[A](val f: Function1[A, Boolean]) extends AnyVal {
+    def apply(v: A) = f(v)
+
+    def &&(g: Function1[A, Boolean]): Function1[A, Boolean] = {
```

Comment:
Personally I find the arrow notation much more explanatory and common for others to understand than `Function1`. It also has similarity to other languages. I.e 

```
def &&(g: A => Boolean): A => Boolean = {
...
```
Is there a reason for using it here in this case? Maybe because of working with generic types?